code/modules/food_and_drinks/recipes/tablecraft/recipes_seafood.dm

```diff
@@ -91,3 +91,29 @@
 	)
 	result = /obj/item/food/nigiri_sushi
 	subcategory = CAT_SEAFOOD
+
+/datum/crafting_recipe/food/meat_poke
+	name ="Meat poke"
+	reqs = list(
+		/obj/item/food/seaweedsheet = 1,
+		/obj/item/food/salad/boiledrice = 1,
+		/obj/item/food/meat/cutlet = 3,
+		/obj/item/food/grown/cabbage = 1,
+		/obj/item/food/tofu = 1,
+		/datum/reagent/consumable/soysauce = 1
+	)
+	result = /obj/item/food/meat_poke
+	subcategory = CAT_SEAFOOD
```

Comment:
Same here, did you mean to make this `CAT_MEAT`?

Reply:
I started from the fact that vegetable sushi (without fish) is in the same category

Reply:
You're right it's like the sushi yeah, then this should be fine